### DISCUSSION TITLE
Valheim adminlist.txt

### DISCUSSION THREAD

**Discussion by cunninghamb505** [GameServerManagers/LinuxGSM#3317] [General]
trying to figure out where to add the adminlist.txt, not sure where in which folder it should sit? any ideas, thank you

**Comment by EugeneGGHQ** [comment]
Assuming you used the default install path, the adminlist.txt is placed at:

`/home/%servername%/.config/unity3d/IronGate/Valheim`

This should be alongside the permitted list and banned list. 

_Note that the '.config' is a hidden directory, so won't appear when normally listing files and folders at command line. _

**Reply by cunninghamb505** [reply]
thank you ill try that out
